File: Shared/Students.Models/ReferenceModels/FinancingType.cs
namespace Students.Models.ReferenceModels;

/// <summary>
/// Тип финансирования
/// </summary>
public class FinancingType
{
  /// <summary>
  /// Id типа финансирования
  /// </summary>
  public Guid Id { get; set; }

  /// <summary>
  /// Имя типа финансирования
  /// </summary>
  public string? SourceName { get; set; }
}

File: Client/src/storage/catalogConfigs/financingType.js
import {
    useGetAllAsync,
    useGetAllPagedAsync,
    useGetOneByIdAsync,
    useAddOneAsync,
    useEditOneAsync,
    useRemoveOneAsync,
} from '../crud/financingTypeCrud.js';
import financingTypeModel from '../models/financingType.js';

export default {
    detailsLink: 'financingType',
    hasDetailsPage: false,
    serverPaged: false,
    properties: financingTypeModel,
    crud: {
        useGetAllAsync,
        useGetAllPagedAsync,
        useGetOneByIdAsync,
        useAddOneAsync,
        useEditOneAsync,
        useRemoveOneAsync,
    },
    columns: [
        {
            title: 'Тип финансирования',
            dataIndex: 'sourceName',
            key: 'sourceName',
        },
    ],
    dataConverter: (data) => data,
};